rotkehlchen/api/rest.py

```diff
@@ -946,6 +946,29 @@ def user_change_password(
         else:
             return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
+    @require_premium_user(active_check=False)
+    def user_premium_key_remove(self, name: str) -> Response:
+        """Returns successful result if API keys are successfully removed"""
+        result_dict: Dict[str, Any] = {'result': None, 'message': ''}
+        success: bool
+
+        if name != self.rotkehlchen.data.username:
+            result_dict['message'] = f'Provided user "{name}" is not the logged in user'
+            return api_response(result_dict, status_code=HTTPStatus.BAD_REQUEST)
+
+        try:
```

Comment:
You don't need the try/except here. The function you wrote, `del_rotkehlchen_premium()` does not raise any such exception.

Reply:
Done.